packages/design-system-react-native/src/components/Text/README.md

```diff
@@ -76,7 +76,7 @@ Available `FontWeight` options:
 
 - `FontWeight.Regular` (Weight `400`)
 - `FontWeight.Medium` (Weight `500`)
-- `FontWeight.Bold` (Weight `700`)
+- `FontWeight.Bold` (Weight `600`)
```

Comment:
Updating docs. I also did a search for `700` to ensure I updated all instances in the codebase. Only colors and this comprehensive font weight type remain

<img width="775" height="657" alt="Image" src="https://github.com/user-attachments/assets/134d3a04-192f-48eb-a4d1-84a3c679b830" />